eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -2,13 +2,7 @@ parameters:
 - name: PackageName
```

Comment:
Not sure if you want to do it as part of this or not but we should consider renaming this file to have it match the other repos.

Reply:
Yeah, right now there are three different names. I'd like to do it as a separate PR when I consolidate the stage yaml across languages (just to keep the diff small against pre-existing files).